package.json

```diff
@@ -1,27 +1,8 @@
 {
   "name": "dpl-design-system",
   "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^12.0.0",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
-    "eslint-plugin-reload": "^0.3.0",
-    "react": "^17.0.2",
-    "react-docgen-typescript": "^2.1.0",
-    "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "typescript": "^4.1.2",
-    "web-vitals": "^1.0.1"
-  },
-  "engines": {
-    "node": ">=16",
-    "npm": ">=7"
-  },
```

Comment:
The commit comment for this change is:

>  Test
> - removal of engines setting in package.json and adding .nvmrc
> - we are doing this in order to be compatible with the node version of ddb-react

Next time consider improving the headline and adding a few more details about your thought process here. A suggestion might be:

>  Remove of engines setting in package.json
>
> Package managers will not allow installation of packages which require a newer engine than what is available. ddb-react currently requires Node 12 which conflicts with the current package and will prevent installation. Even though this package only requires Node 16 for development NPM & Co has no way to distinguish between development and runtime requirements.
>
> To resolve the situation we end up removing the engine specification from package.json. To compensate we add Node version requirements to `.nvmrc` so it will be picked up more easily here. Otherwise we just have to hope developers follow the README.

Perhaps a bit wordy but the important parts are:

1. Getting in the headline right
2. Making clear the reasons why you made the change in the first place—the way things worked before the change (and what was wrong with that), the way they work now, and why you decided to solve it the way you did

https://cbea.ms/git-commit/#why-not-how is a great resource about commit messages :smile:

Reply:
Thank you @kasperg, I amended this and I shall do better with commit messages in the future.